src/controllers/mainController.ts

```diff
@@ -52,7 +54,10 @@ export default class MainController implements vscode.Disposable {
         }
         if (vscodeWrapper) {
             this._vscodeWrapper = vscodeWrapper;
```

Comment:
We can simplify this with the or operator into `this._vscodeWrapper = vscodeWrapper || new VscodeWrapper()`
Or we could use a ternary operator. But I think we should avoid using if/else for this kind of logic